yellow-paper/docs/data-publication-and-availability/index.md

```diff
@@ -68,28 +85,39 @@ One of the places where the existence of consensus make a difference for the rol
+You could say that the transactions are pre-confirmed until they convince the validating light-client on the host.
 
 ### No-consensus
 
-If there is no explicit consensus for the Rollup, staking can still be utilized for leader selection, picking a distinct sequencer which will have a period to propose a block and convince the validating light-client. The user can as earlier define his own confirmation rules and could decide that if the sequencer acknowledge his transaction, then he sees it as confirmed. This can be done fully on trust, of with some signed message the user could take to the host and "slash" the sequencer for not upholding his part of the deal.
+If there is no explicit consensus for the Rollup, staking can still be utilized for leader selection, picking a distinct sequencer which will have a period to propose a block and convince the validating light-client.
+The user can as earlier define his own confirmation rules and could decide that if the sequencer acknowledge his transaction, then he sees it as confirmed.
+This can be done fully on trust, of with some signed message the user could take to the host and "slash" the sequencer for not upholding his part of the deal.
 
 :::info Fernet
 Fernet lives in this category if you have a single sequencer active from the proposal to proof inclusion stage.
 :::
 
-Common for both consensus and no-consensus rollups is that the user can decide when he deems his transaction confirmed. If the user is not satisfied with the guarantee provided by the sequencer, he can always wait for the block to be included in the host chain and get the guarantee from the host chain consensus rules.
+Common for both consensus and no-consensus rollups is that the user can decide when he deems his transaction confirmed.
+If the user is not satisfied with the guarantee provided by the sequencer, he can always wait for the block to be included in the host chain and get the guarantee from the host chain consensus rules.
 
 ## Data Availability and Publication
 
-As alluded to earlier, I am of the school of thought that Data Availability and Publication is different things. Generally, what is referred to as Availability is merely publication, e.g., whether or not the data have been published somewhere. For data published on Ethereum you will currently have no issues getting a hold of the data because there are many full nodes and they behave nicely, but they could are not guaranteed to be so. New nodes are essentially bootstrapped by other friendly nodes.
+As alluded to earlier, we belong to the school of thought that Data Availability and Publication is different things.
+Generally, what is often referred to as Data Availability is merely Data Publication, e.g., whether or not the data have been published somewhere.
+For data published on Ethereum you will currently have no issues getting a hold of the data because there are many full nodes and they behave nicely, but they could are not guaranteed to be so.
```

Comment:
```suggestion
For data published on Ethereum you will currently have no issues getting a hold of the data because there are many full nodes and they behave nicely, but they are not guaranteed to continue doing so.
```